.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -222,6 +222,19 @@ dev_nightly-a7:
       - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-jmx-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-jmx-arm64
         IMG_DESTINATIONS: agent-dev:nightly-${CI_COMMIT_REF_SLUG}-${CI_COMMIT_SHORT_SHA}-py3-jmx,agent-dev:nightly-${CI_COMMIT_REF_SLUG}-py3-jmx
 
+# deploy nightly build to single-machine-performance
+single_machine_performance-nightly-amd64-a7:
+  extends: .docker_publish_job_definition
+  stage: container_build
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Any reason for this particular `stage`, the jobs around it have `dev_container_deploy`, and the other jobs I see in `container_build` don't seem related.

```suggestion
  stage: dev_container_deploy
```